packages/nextjs/scaffold.config.ts

```diff
@@ -10,7 +10,7 @@ export type ScaffoldConfig = {
 
 const scaffoldConfig = {
   // The networks on which your DApp is live
-  targetNetworks: [chains.optimism],
+  targetNetworks: [chains.sepolia],
```

Comment:
Did you deploy BatchRegistry to sepolia as well?

Reply:
I am unable to even deploy to sepolia now the gas is high right now

Reply:
[RESOLVED]

Reply:
I have made all the changes but still having issue deploying sepolia is too high and when trying to use optimism it's giving me issue i don't understand pls can you help me to deploy it from your side

Reply:
![image](https://github.com/user-attachments/assets/bc38a6d6-d59d-4c0e-abf4-5b34882e8f05)